config/locales/en.yml

```diff
@@ -815,6 +815,10 @@ en:
       you_have_not_yet_accepted_or_rejected_offer: "You have not yet accepted or rejected the offer %{title} you received %{date}."
       you_have_not_yet_accepted_or_rejected_request: "You have not yet accepted or rejected the request %{title} you received %{date}."
       show_thread: "Show conversation"
+    branding:
```

Comment:
Good question! And if we go to that direction, should the onboarding language folder be renamed as _marketing_ or _branding_ or _sharetribe_translations_